src/config.js

```diff
@@ -37,7 +37,11 @@ export const _IS_MULTI_TOKEN = true;
  *   uid: '00'
  * }
  */
-export const _DEFAULT_TOKEN = hathorLib.constants.HATHOR_TOKEN_CONFIG;
+export const _DEFAULT_TOKEN = {
+  name: 'Hathor',
+  symbol: 'HTR',
+  uid: '00',
+};
```

Comment:
Why don't we use the default native token config from the lib here?

Reply:
We can, but since this `_DEFAULT_TOKEN` is meant as a build parameter for a custom wallet I thought we should have the fields explicitly stated here instead of using constants from a dependency.
What do you think?

Reply:
I think we can blend the solution, we can declare a literal object with references to native token from the lib.